migrations/versions/d13d58080d80_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: d13d58080d80
+Revises: e65ffbc90def
+Create Date: 2022-05-11 16:06:51.909817
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'd13d58080d80'
+down_revision = 'e65ffbc90def'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('rh_accounts', sa.Column('org_id', sa.Text()))
+    op.create_unique_constraint('uq_org_id', 'rh_accounts', ['org_id'])
```

Comment:
```suggestion
    op.create_unique_constraint('uq_rh_accounts_org_id', 'rh_accounts', ['org_id'])
```

Refer - https://docs.sqlalchemy.org/en/14/core/constraints.html#configuring-a-naming-convention-for-a-metadata-collection